py/src/call.py

```diff
@@ -12,7 +12,10 @@ def main():
     Loops on stdin, reads json commands from lines, outputs single json as a response.
     Starts by outputting "ready"
     """
-    [_myname, db] = sys.argv
+    if len(sys.argv) != 2:
+        print("usage: call.py [sqlite.db]")
```

Comment:
```suggestion
        print("usage: call.py [sqlite.db]", flush=True)
```

It doesn't have any functional difference, but because these should be already used with every other print, I'd add it here as well. What I remember, print doesn't flush if it adds the newline (like it by default does), it needs to be given in the string for autoflush -- could be there's some config for this like the reconfigure.

Reply:
We don't need the explicit flush there because `sys.exit()` flushes standard streams anyway.

Reply:
I don't think we need it for the `check_cairolang_version()` if either.

All the other places probably need it because `sys.stdout` is block-buffered by default.